docs/components/console/console-troubleshooting/common-pitfalls.md

```diff
@@ -15,5 +15,5 @@ Let's take a closer look at common issues and resolutions within Console.
 ## I cannot connect to Zeebe
 
 - Check if your [API client](../manage-clusters/manage-api-clients.md) has the necessary rights. To interact with Zeebe, the **Scope** `Zeebe` must be set.
-- Check if your credentials are configured correctly. There is a CLI tool that allows you to check the status: [`zbctl`](https://www.npmjs.com/package/zbctl). With the command `zbctl status`, you can read the topology. If this command works, the connection can be established.
+- Check if your credentials are configured correctly. There is a community-supported CLI tool that allows you to check the status: [`zbctl`](https://www.npmjs.com/package/zbctl). With the command `zbctl status`, you can read the topology. If this command works, the connection can be established.
```

Comment:
For credential checking, should we continue to encourage people use the topology endpoint in the C8 REST API? @tmetzke? If so, I'll turn this into a reusable component we can slot in to show people how to do a simple cURL request to replace `zbctl status`.

Reply:
Yep, we should, as far as I'm concerned 👍